features/kata/install_uninstall.feature

```diff
@@ -1,53 +1,53 @@
 Feature: kata related features
   # @author pruan@redhat.com
   # @case_id OCP-36508
-  @admin
-  @destructive
-  Scenario: kata container operator installation
-    Given kata container has been installed successfully in the "openshift-sandboxed-containers-operator" project
-    And I verify kata container runtime is installed into a worker node
+  #@admin
```

Comment:
@valtair1891 @cpmeadors how flaky are these?  If you comment them out, you would need to remove the `automated` field in Polarion as well.

Reply:
There are two ways to get a test out from our CI pipeline.
1. Move your test case from automated to not automated in Polarion.
2. Add a tag '@flaky' for the test case in repo verification-tests or cucushift.

Comment the scripts will cause, the test case was picked up in the test runs, but left there with idle status.

Reply:
Adding a tag `@flaky` is the easiest way to exclude it from CI, the runner job will skip them regardless of their automated status.
The tag`@flaky` indicates we are going to fix these tests later, if it's decided we no longer need these tests, it would be better to just remove these tests from repo and polarion.

Reply:
Just FYI, we tried changing the test to No Automated and some script kept changing them back to automated.  Polairon is our source of truth for tests and what status they are in.  Changing them to Not Automated was the obvious choice but it did not work.  I wish it did.

Reply:
Maybe polarion update script is running somewhere and reverts status?